volumes.go

```diff
@@ -23,19 +24,29 @@ type ListVolumeParams struct {
 	Name string `json:"name,omitempty"`
 }
 
+type VolumeRequest struct {
+	ZonalResourceRequest
+	HREF        string    `json:"href,omitempty"`
+	UUID        string    `json:"uuid,omitempty"`
+	Name        string    `json:"name,omitempty"`
+	SizeGB      int       `json:"size_gb,omitempty"`
+	Type        string    `json:"type,omitempty"`
```

Comment:
You can remove the UUID and HREF attributes from the request.

Reply:
Good catch. Fixed.